android/src/main/java/com/testfairy/react/TestFairyModule.java

```diff
@@ -436,7 +441,42 @@ public void run() {
 			}
 		});
 	}
-	
+
+	@ReactMethod
+	public void attachFile(final String filename, final String content) {
+		if (filename == null) {
+			throw new RuntimeException("Cannot attach file without a name!");
+		}
+
+		runOnUi(new Runnable() {
+			@Override
+			public void run() {
+				Context context = getReactApplicationContext();
+				File outputDir = context.getCacheDir();
+				try {
+					File tfTemp = new File(outputDir, "tfTemp");
```

Comment:
Do you need a directory to hold the file? can we just use a "$cachedir/.temp-testfairy-file-attach" ?

Reply:
Yes, otherwise we would have ugly filenames show on timeline.